robottelo/cli/import_.py

```diff
@@ -0,0 +1,325 @@
+# -*- encoding: utf-8 -*-
+"""
+Usage::
+
+    hammer import [OPTIONS] SUBCOMMAND [ARG] ...
+
+Parameters::
+
+    SUBCOMMAND                    subcommand
+    [ARG] ...                     subcommand arguments
+
+Subcommands::
+
+    activation-key                Import Activation Keys
+                                  (from spacewalk-report activation-keys).
+    all                           Load ALL data from a specified directory
+                                  that is in spacewalk-export format.
+    config-file                   Create puppet-modules from Configuration
+                                  Channel content (from spacewalk-report
+                                  config-files-latest).
+    content-host                  Import Content Hosts
+                                  (from spacewalk-report system-profiles).
+    content-view                  Create Content Views based on local/cloned
+                                  Channels (from spacewalk-export-channels).
+    host-collection               Import Host Collections
+                                  (from spacewalk-report system-groups).
+    organization                  Import Organizations
+                                  (from spacewalk-report users).
+    repository                    Import repositories
+                                  (from spacewalk-report repositories).
+    repository-enable             Enable any Red Hat repositories
+                                  accessible to any Organization
+                                  (from spacewalk-report channels).
+    template-snippet              Import template snippets
+                                  (from spacewalk-report kickstart-scripts).
+    user                          Import Users (from spacewalk-report users).
+
+"""
+from robottelo.cli.base import Base
+from robottelo.common import conf, ssh
+
+
+class Import(Base):
+    """Imports configurations from another Satellite instances."""
+
+    command_base = 'import'
+
+    @staticmethod
+    def prepare_import_data(tar_path=None):
+        """Fetch and uncompress the CSV files from the source."""
+
```

Comment:
Please drop all of the blank lines following docstrings throughout this PR. A good, consistent way to lay things out is as such:

```python
def my_method(self, …):
    """Single-line summary.

    Additional information.

    """
    print('Code starts immediately after docstring ends.')
```